functions.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Utlity functions
+
+# Get system architecture
+function get_arch() {
```

Comment:
It would be better if this used the function declaration found in the other scripts, e.g.,

```
get_arch() {
...
}
```

The only place where we use `function` is in update.sh, which should probably be updated.